base/timing.jl

```diff
@@ -430,6 +430,33 @@ macro allocated(ex)
     end
 end
 
+"""
+    @allocs
```

Comment:
name suggestions welcome, this just felt obvious/easy

Reply:
maybe `@nallocs` similar to `ndigits` ?

Reply:
changed to `@allocations` for now, as it leads to a less bad name in BenchmarkTools.jl (where the corresponding name would be `@ballocations`)

Reply:
😂 "ballocs"

Reply:
Yeah, would have been unfortunate